### ISSUE TITLE
Segmentation fault (probably from large lua debug log)

### ISSUE THREAD

**Issue by neilguertin** [darktable-org/darktable#17894]
### Describe the bug

 When the sefault happened I was trying to crop an image. Starting from lighttable, viewing an image fullscreen, I performed these steps repeatedly:
- d to enter darkroom
- adjust crop
- d to exit darkroom to see the result in lighttable fullscreen
I had done this dozens of times over several images. The segfault happened when pressing d to exit darkroom.

Darktable happened to be running from the terminal with `-d lua` (for unrelated troubleshooting). I had started it last night, and the segfault happened in the morning after I continued working. There was this output printed, then the entire contents of darktable_bt_GI3MX2.txt printed. I have no idea when this output was printed or if it is related to the segfault.
```
BFD: error: /usr/lib/debug/.build-id/cc/afe3dc4ac3456c1d7b9cbeeb21b4dea429eb4b.debug(.debug_info) is too large (0x15f8ad bytes)

warning: Can't read data for section '.debug_info' in file '/usr/lib/debug/.build-id/cc/afe3dc4ac3456c1d7b9cbeeb21b4dea429eb4b.debug'

warning: Section .debug_aranges in /usr/lib/debug/.build-id/cc/afe3dc4ac3456c1d7b9cbeeb21b4dea429eb4b.debug entry at offset 0 debug_info_offset 0 does not exists, ignoring .debug_aranges.
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
0x00007f4ed9db6bd7 in __GI___wait4 (pid=502652, stat_loc=0x0, options=0, usage=0x0) at ../sysdeps/unix/sysv/linux/wait4.c:30
30	../sysdeps/unix/sysv/linux/wait4.c: No such file or directory.

<then entire contents of darktable_bt_GI3MX2.txt>
``` 
 I am attaching both darktable_bt_GI3MX2.txt and afe3dc4ac3456c1d7b9cbeeb21b4dea429eb4b.debug (in the zip file)

### Steps to reproduce

I did not try to reproduce it yet. Steps that caused it described above.

### Logfile | Screenshot | Screencast

[darktable_bt_GI3MX2.txt](https://github.com/user-attachments/files/17923096/darktable_bt_GI3MX2.txt)
[luadebug.zip](https://github.com/user-attachments/files/17923139/luadebug.zip)

### Where did you obtain darktable from?

downloaded from www.darktable.org

### darktable version

4.8.0

### What OS are you using?

Linux

### What is the version of your OS?

Debian

### Are you using OpenCL GPU in darktable?

None

**Comment by wpferguson** [comment]
Did you have any lua-scripts running?

**Comment by neilguertin** [comment]
No. I had https://github.com/darktable-org/lua-scripts installed as well as https://github.com/chrik5/lua-scripts/blob/exiftool/contrib/exiftool_export.lua, but nothing was enabled in script manager.

**Comment by neilguertin** [comment]
It happened again. This time while navigating rapidly through images lighttable fullscreen view. Same logs (except for wait4 error). Copying here again for clarity.

However I noticed that afexxx.debug has not been touched. And its last modified timestamp is from several hours before the first crash.

```
BFD: error: /usr/lib/debug/.build-id/cc/afe3dc4ac3456c1d7b9cbeeb21b4dea429eb4b.debug(.debug_info) is too large (0x15f8ad bytes)

warning: Can't read data for section '.debug_info' in file '/usr/lib/debug/.build-id/cc/afe3dc4ac3456c1d7b9cbeeb21b4dea429eb4b.debug'

warning: Section .debug_aranges in /usr/lib/debug/.build-id/cc/afe3dc4ac3456c1d7b9cbeeb21b4dea429eb4b.debug entry at offset 0 debug_info_offset 0 does not exists, ignoring .debug_aranges.
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
0x00007ff0e2d42156 in ?? () from /lib/x86_64-linux-gnu/libpixman-1.so.0

<Then the entire contents of darktable_bt_3NP6X2.txt>
```
[darktable_bt_3NP6X2.txt](https://github.com/user-attachments/files/17924466/darktable_bt_3NP6X2.txt)
